src/server/api/controllers/profile.controller.js

```diff
@@ -17,6 +17,27 @@ const getStudentsProfile = async (userId) => {
   }
 };
 
+const getAdminsProfile = async (userId) => {
+  /* SQL query to get all the admins 
+  SELECT users.name, roles.name
+  FROM users 
+  join user_roles on users.id = user_roles.user_id 
+  join roles on user_roles.role_id = roles.id
+  where roles.name= "admin";
+  */
+  try {
+    const profile = await knex('users')
+      .select('users.name', 'users.admin_role as adminRole')
+      .join('user_roles', 'users.id', 'user_roles.user_id')
+      .join('roles', 'user_roles.role_id', 'roles.id')
+      .where('users.id', userId)
+      .where('roles.name', 'admin');
+    return profile;
```

Comment:
You should return only one item if found.

Reply:
i will modify